apps/web/composables/useCategoryFilter/useCategoryFilter.ts

```diff
@@ -0,0 +1,89 @@
+import { useRoute } from 'nuxt/app';
+import { AgnosticCategoryTree } from '@plentymarkets/plentymarkets-sdk/packages/sdk/src/getters/agnostic.types';
+import { Filters, GetFacetsFromURLResponse, UseCategoryFiltersResponse } from './types';
+
+const nonFilters = new Set(['page', 'sort', 'term', 'itemsPerPage']);
+const defaultPage = 1;
+const defaultItemsPerPage = 20;
```

Comment:
Andrei just added default values to useProducts for itemsPerPage and default. We should find a place for both composables.